### PR TITLE
nc: Improve title and some examples

### PR DESCRIPTION
**Pull request by jaseemabid**
- [X] The page (if new), does not already exist in the repo.
- [X] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [X] The page has 8 or fewer examples.
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [X] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).

### CODE REVIEW

File: pages/common/nc.md [hunk 1]
```diff
@@ -30,10 +30,10 @@
 
 `nc -q {{timeout}} {{ip_address}}`
 
-- Port scanning:
+- Scan open ports on host:
```
Comment (sbrl) [inline pages/common/nc.md]: Perhaps this will read better?

```suggestion
- Scan the open ports of a specified host:
```

File: pages/common/nc.md [hunk 2]
```diff
@@ -1,8 +1,8 @@
 # nc
 
-> Reads and writes tcp or udp data.
+> Netcat is a versatile utility for working with tcp or udp data.
```
Comment (mebeim) [inline pages/common/nc.md]:
```suggestion
> Netcat is a versatile utility for working with TCP or UDP data.
```

I'd capitalize TCP and UDP here while we're at it.

File: pages/common/nc.md [hunk 3]
```diff
@@ -14,7 +14,7 @@
 
 `nc -w {{timeout_in_seconds}} {{ipaddress}} {{port}}`
 
-- Serve a file:
```
Comment (mebeim) [inline pages/common/nc.md]: I think this is good as it is. The command `nc -l port < file` is not actually sending a file automatically. It is serving it waiting for someone to connect to the specified port.

File: pages/common/nc.md [hunk 4]
```diff
@@ -30,10 +30,10 @@
 
 `nc -q {{timeout}} {{ip_address}}`
 
-- Port scanning:
+- Scan open ports on host:
 
 `nc -v -z {{ip_address}} {{port}}`
 
-- Proxy and port forwarding:
+- Proxy and port forwarding, makes a remote connection on local port and prints response:
```
Comment (mebeim) [inline pages/common/nc.md]: This is incorrect, nothing is printed from `nc -l local_port | nc hostname remote_port`.

I would reword it like this:

```suggestion
- Proxy and port forwarding, forward data from a local TCP port to the given remote host:
```

File: pages/common/nc.md (hunk 3 above)
Comment (agnivade) [reply pages/common/nc.md]: Yeah, let's not change this.

File: pages/common/nc.md (hunk 1 above)
Comment (jaseemabid) [reply pages/common/nc.md]: Thanks, updated.

File: pages/common/nc.md (hunk 2 above)
Comment (jaseemabid) [reply pages/common/nc.md]: Thanks, updated.

File: pages/common/nc.md (hunk 3 above)
Comment (jaseemabid) [reply pages/common/nc.md]: Reverted

File: pages/common/nc.md (hunk 4 above)
Comment (jaseemabid) [reply pages/common/nc.md]: Updated.

**Comment by jaseemabid** [comment]
Updated the patch with a force push, addressed the comments.

**Comment by jaseemabid** [comment]
Could you please merge too @mebeim? I can't.

**Comment by mebeim** [comment]
@jaseemabid two approvals by collaborators are required for merging. I've already approved, so let's wait for @agnivade, @sbrl or even someone else.

File: pages/common/nc.md [hunk 5]
```diff
@@ -30,10 +30,10 @@
 
 `nc -q {{timeout}} {{ip_address}}`
 
-- Port scanning:
+- Scan the open ports of a specified host:
 
 `nc -v -z {{ip_address}} {{port}}`
 
-- Proxy and port forwarding:
+- Proxy and port forwarding, forward data from a local TCP port to the given remote host:
```
Comment (agnivade) [inline pages/common/nc.md]: Maybe we can remove "port forwarding" too just as we did in the earlier example. So perhaps - "Act as proxy and forward data from ..."

Comment (mebeim) [reply pages/common/nc.md]: I agree, looks cleaner.

Comment (jaseemabid) [reply pages/common/nc.md]: Updated.
